### ISSUE TITLE
Can't go back after intent

### ISSUE THREAD

**Issue by timbremer** [oprisnik/navdrawer#1]
Hey, this library is awesome but I have one problem. When I start a new action using an intent (to the playstore for example) I can't go back to the app. I think that's because the entry is still active and when I try to go back the intent starts again. But how can I solve this?
Thank you!

```
if (activeItem == moreApps){
        Intent intent = new Intent(Intent.ACTION_VIEW);
        intent.setData(Uri.parse("market://search?q=pub:PubId"));
        startActivity(intent);

    }
```

**Comment by oprisnik** [comment]
Hi!
That is strange. It works for me with the following code in my Activity:

``` java
    private static final NavDrawerEntry ENTRY_MORE_APPS = new NavDrawerEntry.Builder()
            .setTitleResId(R.string.more_apps)
            .setIconResId(R.drawable.abc_ic_go_search_api_mtrl_alpha)
            .setLaunchDelayed(false)
            .build();

    @Override
    public void onEntrySelected(NavDrawerEntry entry) {       
        if (entry == ENTRY_MORE_APPS) {
            Intent intent = new Intent(Intent.ACTION_VIEW);
            intent.setData(Uri.parse("market://search?q=pub:PubId"));
            startActivity(intent);
        } else {
            mSelected = entry;
        }
    }
```

If you use `setLaunchDelayed(false)`, the click will be registered immediately and the selected item will not be highlighted.

It also works for my example app if I do not call `setLaunchDelayed(...)`. In this case the entry will be highlighted though, but if I press back or the up arrow, I get back to the navdrawer app.
I will add an API though, like `isExternal(true)` so that it is more clear (and so that the launch delay has no influence on the highlighting)!

I'll update the sample app and library tomorrow. I hope that it works for you, if not it would be great if you could post the code of your whole sample activity so that I can test it.
Thanks!

**Comment by oprisnik** [comment]
I've just released v1.0.2. Now you can use `.setExternal(true)` to prevent the navigation drawer item from being highlighted.
The following code works for me (taken from the sample application):

``` java
private static final NavDrawerEntry ENTRY_MORE_APPS = new NavDrawerEntry.Builder()
                .setTitleResId(R.string.more_apps)
                .setIconResId(R.drawable.abc_ic_search_api_mtrl_alpha)
                .setExternal(true)
                .build();

...

@Override
public void onEntrySelected(NavDrawerEntry entry) {
        if (entry == ENTRY_MORE_APPS) {
                Intent intent = new Intent(Intent.ACTION_VIEW);
                intent.setData(Uri.parse("market://search?q=pub:Alexander Oprisnik"));
                startActivity(intent);
        } else {
                ...
        }
}
```
